[2024-04-23T19:57:29.187+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-23T19:57:29.200+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_report.transform manual__2024-04-23T19:57:27.600119+00:00 [queued]>
[2024-04-23T19:57:29.205+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_report.transform manual__2024-04-23T19:57:27.600119+00:00 [queued]>
[2024-04-23T19:57:29.205+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-23T19:57:29.213+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): transform> on 2024-04-23 19:57:27.600119+00:00
[2024-04-23T19:57:29.218+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2775) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-23T19:57:29.219+0000] {standard_task_runner.py:63} INFO - Started process 2777 to run task
[2024-04-23T19:57:29.219+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'monthly_report', 'transform', 'manual__2024-04-23T19:57:27.600119+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/homework2.py', '--cfg-path', '/tmp/tmpug260anf']
[2024-04-23T19:57:29.220+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask transform
[2024-04-23T19:57:29.251+0000] {task_command.py:426} INFO - Running <TaskInstance: monthly_report.transform manual__2024-04-23T19:57:27.600119+00:00 [running]> on host 96e40b232902
[2024-04-23T19:57:29.307+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_report' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2024-04-23T19:57:27.600119+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-23T19:57:27.600119+00:00'
[2024-04-23T19:57:29.309+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-23T19:57:29.309+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-23T19:57:29.309+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: transfrom() missing 1 required positional argument: 'date'
[2024-04-23T19:57:29.315+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=monthly_report, task_id=transform, execution_date=20240423T195727, start_date=20240423T195729, end_date=20240423T195729
[2024-04-23T19:57:29.322+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 8 for task transform (transfrom() missing 1 required positional argument: 'date'; 2777)
[2024-04-23T19:57:29.362+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-04-23T19:57:29.372+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-23T19:57:29.373+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
